website/src/components/BasicInstall.js

```diff
@@ -36,7 +37,15 @@ export default function BasicInstall(props){
             </code>
           </TabItem>
 
+          <TabItem value="gha">
+            <p>Add the <a href="https://github.com/VirtusLab/scala-cli-setup">scala-cli-setup</a> action to your workflow:</p>
+            <code>
+              steps:
+                - uses: VirtusLab/scala-cli-setup@main
```

Comment:
I think, that Scala CLI should recommended to use `scala-cli-setup` with `coursier/cache-action` to cache some staff. 

```suggestion
                - uses: coursier/cache-action@v6
                - uses: VirtusLab/scala-cli-setup@main
```